Mixpanel/MixpanelInstance.swift

```diff
@@ -235,9 +246,31 @@ open class MixpanelInstance: CustomDebugStringConvertible, FlushDelegate {
             launchOptions?[UIApplicationLaunchOptionsKey.remoteNotification] as? [AnyHashable: Any] {
                 trackPushNotification(notification, event: "$app_open")
             }
-        #endif
+        #endif // DECIDE
     }
+    #else
+    init(apiToken: String?, flushInterval: Double, name: String) {
+        if let apiToken = apiToken, !apiToken.isEmpty {
+            self.apiToken = apiToken
+        }
 
+        self.name = name
+
+        flushInstance = Flush(basePathIdentifier: name)
```

Comment:
if i do so, all the params that are in an intermediate method would have to become not consts but rather vars (let vs var) and i would hate to remove the immutability for them